subscriptions/models.py

```diff
@@ -58,7 +60,7 @@ def get_expected_next_sequence_number(self, end_date=None):
             end_date = now()
         set_max = self.messageset.get_messageset_max(self.lang)
         runs = self.schedule.get_run_times_between(self.created_at, end_date)
-        count = len(runs)
+        count = len(runs) + (self.initial_sequence_number - 1)
```

Comment:
I add the original sequence to the total run times, so run times between dates + how many run times already passed on this subscription. When `initial_sequence_number` is 1, the default, it has done 0 runs.